radish/main.py

```diff
@@ -76,7 +76,9 @@ def run_features(core):
     # scenario choice
     amount_of_scenarios = sum(len(f.scenarios) for f in core.features_to_run)
     if world.config.scenarios:
-        world.config.scenarios = [int(s) for s in world.config.scenarios.split(",")]
+        world.config.scenarios = [
+            int(s.strip().replace("=", "")) for s in world.config.scenarios.split(",")
```

Comment:
Does this also happen for docopt?

Reply:
If you mean that it breaks if you use `-s=1` the answer is yes (if not then I don't get your question)